### PR TITLE
Moved query params to body

### PR DESCRIPTION
**Pull request by CarmeloCalabro**
#### List of Changes

- Moved query parameters to body

#### How Has This Been Tested?
Tested with Postman

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by AleDore** [comment]
@CarmeloCalabro It seems we must fix linter issue :)

File: openapi/index.yaml [hunk 1]
```diff
@@ -15,117 +15,35 @@ schemes:
...
-        default: 100
-    get:
+    post:
       operationId: getOnlineMerchants
       summary: List of online merchants
       description: |
         List of online merchants alphabetically ordered
+      parameters:
+        - in: body
+          name: searchParameter
```
Comment (AleDore) [inline openapi/index.yaml:25]: we can safely remove name so the entire body maps over the entire request

File: openapi/index.yaml [hunk 2]
```diff
@@ -15,117 +15,35 @@ schemes:
...
-        default: 100
-    get:
+    post:
       operationId: getOfflineMerchants
       summary: List of merchants with a physical address
       description: |
         List of merchants with physical address, ordered by distance from the user by default
+      parameters:
+        - in: body
+          name: searchParameter
```
Comment (AleDore) [inline openapi/index.yaml:43]: ditto :)

File: openapi/index.yaml (hunk 1 above)
Comment (CarmeloCalabro) [reply openapi/index.yaml]: Removed

**Review by AleDore — APPROVED** [review]
@ltronky @CarmeloCalabro : we must add other 2 fields in `OfflineMerchantSearchRequest` that maps the center of the shown map in app

**Comment by AleDore** [comment]
> @ltronky @CarmeloCalabro : we must add other 2 fields in `OfflineMerchantSearchRequest` that maps the center of the shown map in app

We can do it into another PR :)

**Comment by CarmeloCalabro** [comment]
> @ltronky @CarmeloCalabro : we must add other 2 fields in `OfflineMerchantSearchRequest` that maps the center of the shown map in app

Yes, we will create another PR. Thanks

**Pull request merged**